### DISCUSSION TITLE
Customize GettingStarted panel items

### DISCUSSION THREAD

**Discussion by demon** [grafana/grafana#38395] [Dashboards, Panels & Visualizations]
**What would you like to be added**: The ability to add/remove items from the GettingStarted panel.

**Why is this needed**: Having a new user panel is a great thing for onboarding users -- I love the prominent "Fundamentals" and "Create your first dashboard." Very encouraging for a new user. However, I'd like to remove the items for adding Datasources, Plugins, and Users/Teams -- the first two are automatically provisioned in our infrastructure and the last one is irrelevant as we use OIDC for authentication so nobody is creating users.

See also #21881, but that's about disabling it outright.

**Comment by momikiii** [comment]
So...How to remove gettingstarted panel?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
